src/jit/core_templates.expr

```diff
@@ -1436,6 +1436,20 @@
       (carg $1   ptr)
       (carg $2   ptr)) ptr_sz))
 
+(template: neg_I
+  (call (^func &MVM_bigint_neg)
+    (arglist
+      (carg (tc) ptr)
+      (carg $2   ptr)
+      (carg $1   ptr)) int_sz))
```

Comment:
One small thing, looks like you accidently put a return size of int_sz instead of ptr_sz.

I don't think it makes a difference here since the size of a pointer and MVMint64 are the same on x64, but I suppose it could be an issue on other potentially JITted architectures where that is not the case.